test/test-utils.ts

```diff
@@ -0,0 +1,19 @@
+/*!
+ * Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved.
+ */
+import * as fs from 'fs';
+import * as path from 'path';
+import * as rm from 'rimraf';
+
+export function getPathPrefix(dir: string, file: string): string {
```

Comment:
I think the name of the function could be improved. If I understood it correctly, this function gives you a path under which a folder is created in which a test can write stuff isolated from different tests.
Assuming my understanding is correct, maybe a more specific name like `testOutputPath` or `testOutputPathFolder` can help to convey the use of this function.